### ISSUE TITLE
Is this still being updated? 

### ISSUE THREAD

**Issue by hs-ctoth** [jgallagher/amiquip#46]
Several dependencies are out of date, such as amq-protocal, now at 7.1.0, is still at 1.4 in this codebase. My compiler is complaining about old dependencies. 

Please fix, or let us know if you do not intend to, so we can find a different crate to use. 

Thank you.

**Comment by jgallagher** [comment]
> Is this still being updated?

Not really! I haven't been in a position where I'm using this crate in quite a while, and currently do not have time to maintain it. That could change in the future, but I don't know when that would be. If old dependencies are a problem, I'd suggest either forking (and opening a PR if you'd like, although the same caveat goes - I'm currently very slow to look at them) or using a different crate.

**Comment by hs-ctoth** [comment]
Thanks for getting back to me so quickly!
